rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.1.0-release-notes.adoc

```diff
@@ -0,0 +1,34 @@
+= RPA Recorder 6.1.0 Release Notes
+
+RPA Recorder is MuleSoft’s tool for recording a business process for automation.
+
+== TBD
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The new attributes selection panel enables you to fine-tune the XPath for web elements.
+* You can now capture _Mouse Over_ events using the `F2` key.
```

Comment:
+ "while recording Web Tasks" - also this feature was added for the Web Task recording, previously it was only available for Bot Tasks.